src/components/NoteCard.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <div v-bind:class="{ textIndsidePadding: homePage, textInside: !homePage }">
+      <h1>{{ note.get("title") }}</h1>
+      <p class="time">
+        {{
+          note.createdAt.toLocaleString("en-US", {
+            year: "numeric",
+            month: "numeric",
+            day: "numeric",
+            hour: "numeric",
+            minute: "numeric"
+          })
+        }}
+      </p>
+      <p>{{ note.get("content") }}</p>
+      <ul>
+        <router-link
+          v-for="client in note.get('clients')"
+          :key="client.id"
+          :to="`/client/${client.id}`"
+          class="client"
+        >
+          <li>@{{ client.get("fullName") }}</li>
+        </router-link>
+      </ul>
```

Comment:
Lemme know if I should use the bullet points. I thought it looked more organized this way.

Reply:
Maybe the indentation can be reduced a bit.

Also, maybe explore Font Awesome's icon library to find an icon to replace the bullet point (and the `@` symbol)?